Source/Libs/SteamVdfParse.hpp

```diff
@@ -9,17 +9,28 @@
 
 #define MAGIC_VALUE 0x07564426
 
+//! Steam VDF Parsing
+/*! A bunch of utility functions to parse the Steam VDF binary format
+*/
 namespace SteamVdfParse
 {
 
 namespace pt = boost::property_tree;
     
+/*! Function to translate an istream object into a 32-bit little endian integer
```

Comment:
Does not "translate" an istream, it reads in the integers as big endian and converts them to little endian.